addons/mail/controllers/discuss.py

```diff
@@ -573,3 +573,29 @@ def mail_guest_update_name(self, guest_id, name):
         if guest_to_rename_sudo != guest and not request.env.user._is_admin():
             raise NotFound()
         guest_to_rename_sudo._update_name(name)
+
+    # --------------------------------------------------------------------------
+    # Link preview API
+    # --------------------------------------------------------------------------
+
+    @http.route('/mail/link_preview', methods=['POST'], type='json', auth='public')
+    def mail_link_preview(self, message_id, urls):
+        throttle = request.env['ir.config_parameter'].sudo().get_param('mail.link_preview_throttle', 99)
+        if int(throttle) == 0:
```

Comment:
Same remark here for variable name and condition.
Also now that it is used in several places I would consider moving this code as a model method.